apps/snitch_core/priv/repo/migrations/20180307121608_add_unique_index_to_country.exs

```diff
@@ -0,0 +1,10 @@
+defmodule Snitch.Repo.Migrations.AddUniqueIndexToCountry do
+  use Ecto.Migration
+
+  def change do
+    create unique_index(:snitch_countries, [:iso])
+    create unique_index(:snitch_countries, [:iso3])
+    create unique_index(:snitch_countries, [:name])
+    create unique_index(:snitch_countries, [:numcode])
```

Comment:
you can create unique index for multiple cols at once,
give it a look https://stackoverflow.com/questions/36418223/creating-a-unique-constraint-on-two-columns-together-in-ecto

Reply:
But we don't want to index on multiple columns because a multi-column index would allow duplicate `:iso` as long as others columns are different.